source/boards/get-started-with-boards.rst

```diff
@@ -0,0 +1,78 @@
+Get Started with Mattermost Boards
+==================================
+
+Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.
+
+.. |product-list| image:: ../images/products_E82F.svg
+  :height: 24px
+  :width: 24px
+  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.
+
+Boards are used for planning projects collaboratively. You can access the list of channels and their boards from the drop-down menu in the left-hand sidebar. Boards are made up of cards, and cards can have a wide range of purposes.
+
+See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Channels to enable team collaboration.
```

Comment:
would "Channels" here refer to the product vertical or the communication vehicle? If the latter, lowercase "c" :)